agent/api/task/task.go

```diff
@@ -2224,14 +2225,8 @@ func (task *Task) GetID() (string, error) {
 		return "", errors.Errorf("task get-id: malformed task resource: %s", resource)
 	}
 
-	resourceSplit := strings.SplitN(resource, arnResourceDelimiter, arnResourceSections)
```

Comment:
i think `arnResourceSections` is unused now? can you remove it

Reply:
That should have been caught by our new static analysis check. @tamor1 you should sync your local dev branch with the aws/ upstream and base your commits off of a branch. Currently it looks like you've made your commits off your local master branch.